test/versioned/bunyan/bunyan.tap.js

```diff
@@ -0,0 +1,315 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const tap = require('tap')
+const helper = require('../../lib/agent_helper')
+const { validateLogLine } = require('../../lib/logging-helper')
+const { LOGGING } = require('../../../lib/metrics/names')
+// winston puts the log line getting construct through formatters on a symbol
```

Comment:
you have a bit of copy pasta in here. i've seen a few winston strings 😄

Reply:
you caught me! :laughing: 

(I'll clean it up)